lib/rubocop/cop/style/data_inheritance.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for inheritance from Data.define.
```

Comment:
I'd also mention here what's the problem we're trying to solve (avoid the creation of the anonymous parent class).

Reply:
Added couple words, thanks for the comment! I was considering adding outputs of `.ancestors` and whatnot, but was afraid it will be too verbose then. WDYT about current form?